modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -62,4 +62,68 @@ test.describe('Manage object fields through Model Builder', () => {
 			ListTypeDefinition.id
 		);
 	});
+
+	test('can delete object field', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+	}) => {
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				status: {code: 0},
+			});
+
+		await apiHelpers.objectAdmin.postObjectFieldByExternalReferenceCode(
+			objectDefinition.externalReferenceCode,
+			{
+				DBType: 'Integer',
+				label: {
+					en_US: 'intField',
+				},
+
+				listTypeDefinitionId: 0,
+				localized: false,
+				name: 'intField',
+				objectFieldSettings: [],
+				readOnly: 'false',
+				readOnlyConditionExpression: '',
+				required: false,
+				state: false,
+				system: false,
+			}
+		);
+
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await page.setViewportSize({height: 920, width: 1900});
```

Comment:
Hey @rafaellajordao, can you explain to me why you decided to specifically use these values? Thanks!

Reply:
I used this value because it was the one that best adapted to the notebook's screen, but if there is a better one I can make the change!

Reply:
Hey @rafaellajordao, could you try using the `screen.height`, `screen.width` values?

Perhaps with this change, the test can adjust to device screen sizes. 😄